Transfer/Tests/CreateTransferPresenterTests.swift

```diff
@@ -207,13 +211,23 @@ class CreateTransferTests: XCTestCase {
         Hyperwallet.clearInstance()
         HyperwalletTestHelper.programModel = .walletModel
         Hyperwallet.setup(HyperwalletTestHelper.authenticationProvider)
-        mockView.stopOnError = false
-        initializePresenter(createTransferResult: .failure, showAllAvailableSources: true)
-        assertResponse(isShowErrorPerformed: false,
-                       transferSourceCellConfigurationsCount: 3,
-                       transferSourceType: .user,
-                       selectedTransferDestination: true,
-                       isAvailableBalancePresent: true)
+        if error?.group != .business {
+            mockView.stopOnError = true
+            initializePresenter(createTransferResult: .unexpectedFailure, showAllAvailableSources: true)
+            assertResponse(isShowErrorPerformed: true,
+                           transferSourceCellConfigurationsCount: 3,
+                           transferSourceType: .user,
+                           selectedTransferDestination: true,
+                           isAvailableBalancePresent: true)
+        } else {
+            mockView.stopOnError = false
+            initializePresenter(createTransferResult: .failure, showAllAvailableSources: true)
+            assertResponse(isShowErrorPerformed: false,
+                           transferSourceCellConfigurationsCount: 3,
+                           transferSourceType: .user,
+                           selectedTransferDestination: true,
+                           isAvailableBalancePresent: true)
+        }
```

Comment:
why enhance  this? Create a new test case for unexpected error

Reply:
ok